shared/menubar/index.desktop.js

```diff
@@ -370,7 +370,7 @@ const BadgeIcon = ({tab, countMap, openApp}) => {
         color={Styles.globalColors.darkBlue4}
         hoverColor={Styles.globalColors.white}
         onClick={() => openApp(tab)}
-        sizeType="Big"
+        fontSize={20}
```

Comment:
Is this a one-off / do we not want a real `sizeType` for it?

Reply:
it's a one-off.